### ISSUE TITLE
Private IP frontend support

### ISSUE THREAD

**Issue by richardcatania** [Azure/application-gateway-kubernetes-ingress#83]
We are in need of creating an end point which is accessibly only internally.

If we move the associated listeners it automatically creates to the Private IP Address, it momentarily works as expected until kubernetes automatically re-applies the config to the public IP.

**Comment by asridharan** [comment]
@richardcatania were do you mean to say that initially the Application Gateway v2 had a public IP, and when you move it to a private IP the ingress controller overrode that config with the public IP?

**Comment by richardcatania** [comment]
What I meant was that if we configure the Associated Listeners to the Private from the Azure Portal, Kubernetes will move them back to Public. Please have a look at the screen shot below so you can see a clearer picture of what I am trying to explain:

https://pasteboard.co/HSJKHZh.png

**Comment by joeizy** [comment]
Does this support ILB App Gateway deployment at all?

**Comment by benrj** [comment]
I've got the same question.  [The docs](https://azure.github.io/application-gateway-kubernetes-ingress/docs/install-existing.html) seem to indicate that this only works with a Standard V2 application gateway.  However, V2 gateways apparently require you to have a public IP address.  

Our security requirements prevent us from serving up on public IPs.  We've used Standard gateways with private IPs only (with backend pools of VMs) + internal DNS to meet our requirements, but now we're prototyping AKS and wanting to leverage the [App Gateway/Ingress Controller](https://azure.github.io/application-gateway-kubernetes-ingress), but it appears we won't be able to.

Can standard gateways be used? Or is there another workaround for this?
